packages/breadcrumbs/src/styled/StyledBreadcrumbItem.spec.tsx

```diff
@@ -6,19 +6,27 @@
  */
 
 import React from 'react';
+import { stripUnit } from 'polished';
```

Comment:
Use the `lib` imports to help reduce bundle size for consumers. Probably something like `polished/lib/utilities/stripUnit`

Reply:
Awesome, reducing the bundle size sounds good. I'll make that change.

Reply:
`import stripUnit from 'polished/lib/helpers/stripUnit';`

(although spec files don't hit the bundle ;)